_pages/CalDAV/Client-Implementations.md

```diff
@@ -25,7 +25,7 @@ CalConnect only provides a list of known implementations of CalDAV clients. Any
 
 aCal is a CalDAV client for Android which requires a CalDAV server to function.
 
-### [AgenDAV](http://agendav.org/)
+### [AgenDAV](https://agendav.github.io/agendav/)
```

Comment:
It might be better to just link the repo directly, if you'd prefer that option here's the link
```suggestion
### [AgenDAV](https://github.com/agendav/agendav)
```